src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -203,7 +203,7 @@ function updatePhaseTableChart(flux) {
             x: {columnOrExpr: LC.PHASE_CNAME, options: 'grid'},
             y: {columnOrExpr: flux, options: 'grid,flip'}
         };
-        loadXYPlot({chartId: LC.PHASE_FOLDED, tblId: LC.PHASE_FOLDED, xyPlotParams});
+        loadXYPlot({chartId: LC.PHASE_FOLDED, tblId: LC.PHASE_FOLDED, xyPlotParams, help_id: 'main1TSV.plot'});
```

Comment:
This would require a major refactoring. Chart is expecting the help_id from the dispatcher. We can talk to @tgoldina and make a separate ticket for the future.